domains/transfer-agreements/TransferAgreements.API/API.IntegrationTests/Controllers/TransferAgreementsControllerTest.cs

```diff
@@ -38,4 +41,41 @@ public async Task Create_ShouldFail_WhenModelInvalid()
         var response = await authenticatedClient.PostAsJsonAsync("api/transfer-agreements", new { });
         response.StatusCode.Should().Be(HttpStatusCode.BadRequest);
     }
+
+    [Fact]
+    public async Task GetBySubjectId_ShouldReturnTransferAgreements_WhenUserHasTransferAgreements()
+    {
+        var sub = Guid.NewGuid().ToString();
+        await factory.SeedData(context =>
```

Comment:
Maybe put this in a seperate method for seeding the db i am sure that this could be needed elsewhere in the test :)